src/main/build/debug.h

```diff
@@ -71,6 +71,7 @@ typedef enum {
     DEBUG_MAX7456_SPICLOCK,
     DEBUG_SBUS,
     DEBUG_FPORT,
+    DEBUG_FPORT2,
```

Comment:
New debug items _MUST_ go to the end of the list

Reply:
Doe